### PR TITLE
[16.0][OU-ADD] hr_contract

### PR DESCRIPTION
**Pull request by remytms**
Most of the changes are managed by the migration script of hr module. Load the noupdate_changes file.

### CODE REVIEW

**Comment by legalsylvain** [comment]
/ocabot migration hr_contract

<dependency></dependency>

**Review by marielejeune — APPROVED** [review]
Looks good

**Review by nguyenvietlam0640 — APPROVED** [review]

**Review by victoralmau — APPROVED** [review]

**Comment by andreampiovesana** [comment]
merge?

**Comment by remytms** [comment]
I think that it will be merge when dependencies will be scripted and merged.

**Comment by remytms** [comment]
@legalsylvain I think this is not blocked by dependency anymore, since the merge of module `hr`.

File: openupgrade_scripts/scripts/hr_contract/16.0.1.0/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Copyright 2023 Coop IT Easy (https://coopiteasy.be)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.load_data(env.cr, "hr_contract", "16.0.1.0/noupdate_changes.xml")
```
Comment (pedrobaeza) [inline openupgrade_scripts/scripts/hr_contract/16.0.1.0/post-migration.py:9]: Analyzing noupdate_changes.xml file, the `group_hr_contract_manager` is being modified through `(4, id)`, which means that the previous content is kept, and we don't want this, so you should add first a `(5, )` command for erasing current content.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by pedrobaeza** [comment]
Superseded by #4336 for adding the missing piece.

**Pull request closed without merging**